moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -54,6 +54,13 @@ namespace moveit_setup_assistant
 // File system
 namespace fs = boost::filesystem;
 
+std::string toString(double val)
+{
+    std::stringstream ss("");
+    ss << val;
+    return ss.str();
+}
+
```

Comment:
Why not just use `std::to_string()`?

Reply:
`std::to_string()` unfortunately is a misnomer, it should be called `std::to_localized_string()` as it will give me (`LANG=de_DE`) an output of `1,23` instead of the machine readable and expected `1.23` as it uses the c functions internally that obey environment variables.

`std::stringstream` is more likely not to use localized strings but for safety https://github.com/ros-planning/moveit/blob/master/moveit_core/utils/src/lexical_casts.cpp
includes setting the `imbue`: https://github.com/ros-planning/moveit/blob/3a61f184b82d01c75333292ca3e12ecd2765ecde/moveit_core/utils/src/lexical_casts.cpp#L51

What this should really be doing is to use the `moveit::core::to_string()` from `#include <moveit/utils/lexical_casts.h>`

Reply:
Thanks for this, I was looking for an alternative to this function.